traffic_ops/testing/api/v5/servers_hostname_update_test.go

```diff
@@ -125,9 +137,15 @@ func validateServerApplyTimes(hostName string, expectedResp map[string]interface
 				case "ConfigApplyTime":
 					assert.RequireNotNil(t, resp.Response[0].ConfigApplyTime, "Expected ConfigApplyTime to not be nil.")
 					assert.Equal(t, true, server.ConfigApplyTime.Equal(expected.(time.Time)), "Expected ConfigApplyTime to be %v, but got %v", expected, server.ConfigApplyTime)
+				case "ConfigUpdateFailed":
+					assert.RequireNotNil(t, resp.Response[0].ConfigUpdateFailed, "Expected ConfigUpdateFailed to not be nil.")
```

Comment:
this check cannot possibly fail because non-reference fields can never be `nil`

Reply:
`ServerUpdateStatusV5` still has this as a pointer since it's an optional field.